vms/platformvm/validator_set_property_test.go

```diff
@@ -94,7 +96,7 @@ func TestGetValidatorsSetProperty(t *testing.T) {
 			}
 
 			validatorsSetByHeightAndSubnet := make(map[uint64]map[ids.ID]map[ids.NodeID]*validators.GetValidatorOutput)
-			if err := takeValidatorsSnapshotAtCurrentHeightAndTest(vm, validatorsSetByHeightAndSubnet); err != nil {
+			if err := takeValidatorsSnapshotAtCurrentHeight(vm, validatorsSetByHeightAndSubnet); err != nil {
```

Comment:
taking snapshot here but testing height over the full range, at the end of this test, rather than specific heights